### ISSUE TITLE
Radio buttons when grading your peers need labels

### ISSUE THREAD

**Issue by sabahtester** [amandadoughty/moodle-mod_peerwork#52]
The radio boxes in the 'grade your peers' section do not have clear labels and therefore, NVDA does not read out the name of the students or the headers for the radio boxes. Therefore, a screen reader user would not be aware of who they are grading and what they are selecting on the scale. Pasted from NVDA speach viewer:

radio button  checked  1 of 3
radio button  checked  2 of 3
radio button  not checked  1 of 2
radio button  not checked  1 of 2
radio button  not checked  1 of 2
radio button  not checked  1 of 2
